addons/pharma/stringtable.xml

```diff
@@ -2754,7 +2756,7 @@
             <Italian>Possibilità di ostruzione della IV senza prima aver svolto uno sciacquo salino</Italian>
             <French>Chance de blocage des IV dû à une administration intra-veineuse de médicaments sans rinçage à la solution saline</French>
             <Korean>식염수 주입 없이 약물 투여로 인한 IV 막힘 확률</Korean>
-            <German>Möglichkeit einer Verstopfung der IV durch Medikamentengabe ohne Kochsalzspülung</German>
+            <German>Möglichkeit einer Okklusion der IV durch Medikamentengabe ohne Kochsalzspülung</German>
```

Comment:
Es ist eine Vertsopfung des IV. Also würde ich das auch dabei belassen :D

Reply:
is das nich von dir?

Reply:
Kann sein.. bitte ändere das um :D